packages/devextreme/js/__internal/scheduler/appointments/m_appointment_collection.ts

```diff
@@ -597,7 +607,7 @@ class SchedulerAppointments extends CollectionWidget {
         data: rawAppointment,
         groupIndex: settings.groupIndex,
         groupTexts: getGroupTexts(groups, groupsLeafs, resourceById, settings.groupIndex),
-        observer: this.option('observer'),
+        scheduler: this.option('scheduler'),
```

Comment:
If we use scheduler as a prop, then we don't avoid of boilerplate.
Let's use `WidgetNotifyScheduler` as a prop for m_appointment, m_appointment_collection and m_workspace then and remove all boilerplate.